src/components/dashboard/Dashboard.jsx

```diff
@@ -14,14 +14,23 @@ import { DashboardSidebar } from './sidebar';
 import SubscriptionExpirationModal from './SubscriptionExpirationModal';
 import { UserSubsidyContext } from '../enterprise-user-subsidy';
 
-export const LICENCE_ACTIVATION_MESSAGE = 'Your license has been successfully activated.';
+export const LICENCE_ACTIVATION_MESSAGE = 'Your license was successfully activated.';
 
 export default function Dashboard() {
   const { enterpriseConfig } = useContext(AppContext);
   const { subscriptionPlan, showExpirationNotifications } = useContext(UserSubsidyContext);
   const { state } = useLocation();
+  const history = useHistory();
   const [isActivationAlertOpen, , closeActivationAlert] = useToggle(!!state?.activationSuccess);
 
+  useEffect(() => {
+    if (state?.activationSuccess) {
+      const updatedLocationState = { ...state };
+      delete updatedLocationState.activationSuccess;
+      history.replace({ ...history.location, state: updatedLocationState });
+    }
+  }, []);
```

Comment:
The activation alert currently persists on page refresh, which is unexpected. If `state.activationSuccess` is set, this clears it out so it doesn't appear again.